e2e/app/page/workspace-analysis-page.ts

```diff
@@ -19,7 +19,7 @@ export default class WorkspaceAnalysisPage extends WorkspaceBase {
   }
 
   async isLoaded(): Promise<boolean> {
-    await Promise.all([waitForDocumentTitle(this.page, PageTitle), this.createNewNotebookLink().waitUntilEnabled()]);
```

Comment:
removed this because the "create new notebook" button is disabled for Reader and it was causing the notebook-reader-actions.spec.ts test to fail.